ts/src/binance.ts

```diff
@@ -4280,8 +4284,10 @@ export default class binance extends Exchange {
         const market = this.market (symbol);
         // binance docs say that the default limit 500, max 1500 for futures, max 1000 for spot markets
         // the reality is that the time range wider than 500 candles won't work right
-        const defaultLimit = 500;
-        const maxLimit = 1500;
+        let defaultLimit: Num;
```

Comment:
my question is,  instead of using this terminology should we use the unified `limit` keyword? so users can provide it as an argument or option and if they don't we use these defaults.